doc/install.rst

```diff
@@ -28,6 +28,51 @@ There are different ways to install scikit-learn:
 Installing the latest release
 =============================
 
+Quickstart
+----------
+
+.. raw:: html
+
+  <div class="install">
+       <p><strong>System</strong></p>
+          <input type="radio" name="os" id="quickstart-mac" checked>
+          <label for="quickstart-mac">macOS / OSX</label>
+          <input type="radio" name="os" id="quickstart-win">
+          <label for="quickstart-win">Windows</label>
+          <input type="radio" name="os" id="quickstart-lin">
+          <label for="quickstart-lin">Linux</label>
+       <p><strong>Packager</strong></p>
+          <input type="radio" name="packager" id="quickstart-pip" checked>
+          <label for="quickstart-pip">pip</label>
+          <input type="radio" name="packager" id="quickstart-conda">
+          <label for="quickstart-conda">conda</label>
+          <input type="radio" name="packager" id="quickstart-source">
+          <label for="quickstart-source">source</label>
+       <p><strong>Pip Virtual Environment</strong></p>
+          <input type="checkbox" name="config" id="quickstart-venv">
+          <label for="quickstart-venv">Virtual environment</label>
```

Comment:
There could be a single checkbox "Use virtualenv / conda environment"  display the `python -m venv sklearn-venv` or  `conda create -n sklearn-env` command depending on the value of the `packager` radio button.

Reply:
Maybe change the OS order by popularity: "Windows", "macOS", "Linux (and other POSIX systems)"

Reply:
Done in b8f44c4023a0c5eb1d8a6107defed15bcf4d1166